code/lstm.py

```diff
@@ -407,8 +414,10 @@ def train_lstm(
     print 'Loading data'
     train, valid, test = load_data(n_words=n_words, valid_portion=0.05,
                                    maxlen=maxlen)
+    if test_size > 0:
+        test = (test[0][:test_size], test[1][:test_size])
```

Comment:
Now that the data is sorted by length, it means the test set only consists in the shortest sequences, which is biased.